src/index.ts

```diff
@@ -96,22 +96,23 @@ Documentation:
   process.exit(0);
 }
 
-const USER_FUNCTION = getUserFunction(CODE_LOCATION, TARGET);
-if (!USER_FUNCTION) {
-  console.error('Could not load the function, shutting down.');
-  // eslint-disable-next-line no-process-exit
-  process.exit(1);
-}
+getUserFunction(CODE_LOCATION, TARGET).then(USER_FUNCTION => {
```

Comment:
nit: I think the intended convention in this class was to use `UPPER_CASE` for constants. Could we convert this to `camelCase` now that is a mutable parameter:

```typescript
getUserFunction(CODE_LOCATION, TARGET).then(userFunction => {
```